pyecasound/ecacontrol.py

```diff
@@ -1,291 +1,277 @@
-"""Native python ECI (ecasound control interface) implementation
```

Comment:
@marclava Can you update git commit message to match style of previous ne. Add "pyecasound: " prefix. Short message summarizing what this has is enough.

This is mostly style changes to match more modern python3 style, right?